src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const rules = sourceString.split(';');
+
+  const result = {};
+
+  rules.forEach((rule) => {
+    const ruleCopy = rule.trim();
```

Comment:
[CODE KNOWLEDGE] - Don't need to make a copy before each operation. Since `rule` is not being modified, there's no need to create `ruleCopy`; you can directly perform operations on `rule`.